spacecmd/src/spacecmd/package.py

```diff
@@ -43,9 +43,21 @@
 except AttributeError:
     _ = translation.gettext
 
+
+def print_help_advanced_search(command, description):
+    print('%s: %s' % (command, description))
+    print(_('usage: %s NAME|QUERY' % command))
+    print('')
+    print(_('Example: %s kernel' % command))
+    print('')
+    print(_('Advanced Search:'))
+    print(_('Available Fields: name, epoch, version, release, arch, description, summary'))
+    print(_('Example: name:kernel AND version:2.6.18 AND description:devel'))
```

Comment:
Almost Identical help text is already present in function `help_package_search`, why are you creating new function instead of adapting the existing one and use that?

Reply:
@aaannz I've now updated this PR to reuse `help_package_search` instead of creating a new function. Please take another look.